api.swagger.yaml

```diff
@@ -5471,6 +5471,16 @@ paths:
           description: |
             The notification does not exist.
       description: Mark a notification as read.
+  '/notifications/clear_all':
+    delete:
+      tags:
+        - notifications
+      responses:
+        '200':
+          description: The user marked the notification as read.
+        '400':
```

Comment:
Do we know of any error responses that we can expect to be more specific?

Reply:
Not as of yet. I will probably update onces i have finished all tasks in the epic.

Reply:
Can any of these be added?